gin/handleAdminViewUsers.go

```diff
@@ -38,3 +38,20 @@ func handleAdminViewUsers(c *gin.Context) {
 // 		"totalOrders": totalOrders,
 // 	})
 // }
+func handleGetUserAndAllOrdersFromUser(c *gin.Context) {
+	var user gorm.User
+	userID, err := strconv.ParseUint(c.Param("userid"), 16, 16)
+	if err != nil {
+		logger.Info("Cannot convert userid to uint")
+		return
+	}
+	//get a list of order object for an user
+	allOrders, err := gorm.GetAllOrderFromUser(uint(userID))
+	//get an user object
+	userInfo, err := user.GetOneUser(uint(userID))
```

Comment:
you can use the function `PopulateByID()` in `user.go`. It basically do the same: put in an ID and it will get an user

Reply:
Da thay =))